src/renderer/patches/keybinds.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import { keybindCallbacks } from "renderer";
+
+import { addPatch } from "./shared";
+
+addPatch({
+    patches: [
+        {
+            find: ".default.Messages.KEYBINDS,children:",
+            replacement: {
+                match: /.\.isPlatformEmbedded/,
+                replace: "true"
+            }
+        },
+        {
+            find: "[kb store] KeybindStore",
+            replacement: {
+                match: /(inputEventRegister\(parseInt\((.{1,2})\),(.{1,2}),(.{1,2}),(.{1,2})\);else\{)([^;]*;[^;]*;.{1,2}\.keyup&&.{1,2}\.bindGlobal\(\(0,(.{1,2}\.toString)\))/,
```

Comment:
you should use `\i` instead of `.{1,2}` to match variables. eslint will yell at you as it wasn't patched here unlike in main vencord; just tell it to shot up with an `// eslint-ignore`